test/wrapCodeInTestFunction.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = function wrapCodeInTestFunction(code) {
+  return 'it(\'should test\', async function test() { ' + code + '; })';
+};
```

Comment:
1. Since we're getting some refactoring in, more template strings!
2. I prefer to declare exports separately.

```suggestion
function wrapCodeInTestFunction(code) {
  return `it('should test', async function test() { ${code}; })`;
};

module.exports = wrapCodeInTestFunction;
```